imagebuildah/stage_executor.go

```diff
@@ -668,6 +668,17 @@ func (s *StageExecutor) prepare(ctx context.Context, from string, initializeIBCo
 		for _, p := range builder.Ports() {
 			ports[docker.Port(p)] = struct{}{}
 		}
+		buildahHealthConfig := builder.Healthcheck()
+		var dockerHealthConfig *docker.HealthConfig
```

Comment:
Nit: Can you leave out the "docker" adjective.  Just call this healthCheck 
var healthCheck *docker.HealthConfig